docs/howto.rst

```diff
@@ -101,3 +101,18 @@ access as `requests cookies <http://requests.readthedocs.org/en/latest/user/quic
     :lines: 7-20
 
 Full example: :download:`using_cookies.py <examples/using_cookies.py>`
+
+
+Certificates
+------------
+
+SSL certificates can be set by passing an instance of any class which
```

Comment:
This wording is a bit confusing.  It's SSL *trust roots* which can be set; setting a "certificate" implies client-certificate authentication, which still is in fact not supported.